_data/cloud.yml

```diff
@@ -186,4 +186,4 @@ websites:
       tfa: yes
       software: yes
       hardware: yes
-      doc: https://www.vultr.com/docs/using-two-factor-authentication-to-login-to-vultr-control-panel
+      doc: https://www.vultr.com/docs/using-two-factor-authentication-to-login-to-vultr-control-panel
```

Comment:
As per our `.editorconfig`, it contains `insert_final_newline = true`.
Please respect that.